prawns/governance/schema.graphql

```diff
@@ -22,7 +22,7 @@ type SubstrateGovernanceAccount @entity {
   totalTreasurySpendProposals: Int!
   councilVotes: [SubstrateCouncilVote]! @derivedFrom(field: "account")
   proposalVotes: [SubstrateProposalVote]! @derivedFrom(field: "account")
-  proposalSeconds: [SubstrateProposalSecond]! @derivedFrom(field: "account")
+  democracyProposalSeconds: [SubstrateDemocracyProposalSecond]! @derivedFrom(field: "account")
```

Comment:
Regarding the renames here, should we actually duplicate the fields, having room to deploy these changes, let the clients migrate (native app) and then do another PR where we do the cleanup?

Reply:
Just thinking, maybe just synching with them is just more practical but we need to accept some downtime :)

Reply:
But they are wrapped in the Graph:
https://graph.litentry.io/graphql?query=query%20MyQuery%20%7B%0AsubstrateDemocracyProposals%7B%0A%20%20account%20%7B%0A%20%20%20%20totalProposalSeconds%0A%20%20%20%20proposalSeconds%7B%0A%20%20%20%20%20%20account%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%7D

☝️ When you mean directly you also mean: "not even from the graph"?

Reply:
I mean directly from the mobile apps.. The only data we are fetching from the mobile apps that is coming from the indexed data are the tips, and we are doing it with a custom resolver, so no schema stitching